sdk/search/Microsoft.Azure.Management.Search/tests/Microsoft.Azure.Management.Search.Tests.csproj

```diff
@@ -8,10 +8,19 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <ExcludeFromBuild>true</ExcludeFromBuild>
```

Comment:
@chidozieononiwu is there a reason you added these properties to exclude?

Reply:
There are failures in the test as it is not able to build for both net461 and netcore2.0 am looking into it. And I will file an issue if I cannot immediately resolve it.